build.gradle

```diff
@@ -57,8 +57,8 @@ apply plugin: 'osgi'
 apply plugin: 'version-injection'
 apply plugin: 'eclipse'
 
-targetCompatibility = "1.7"
-sourceCompatibility = "1.7"
+targetCompatibility = "1.8"
```

Comment:
it's not required if we use the same value for both source and target... according to [gradle docs](https://docs.gradle.org/current/userguide/java_plugin.html#_other_convention_properties), target = source by default.

Reply:
@sskorol - Thanks for calling that out. I have fixed that as well, to just include `sourceCompatibility`